specs/appwindow-spec.md

```diff
@@ -0,0 +1,68 @@
+Window.AppWindow api
+===
+New api to simplify accessing [appwindow](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/windowing/windowing-overview) functionality through WinUI code
+
+# Background
+AppWindow provides good set of functionalities for using HWNDs than dealing with win32 apis directly. It is already accessible to winui desktop apps via winappsdk runtime. Currently, users must write a lot of boiler plate code to get to appwindow object and use it. 
+
+[Xaml Window](https://docs.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.window) will expose AppWindow object directly to app developer through an api. Instead of writing a lot of boiler plate code everywhere, app developer can use this api, reducing code bloat, and making appwindow apis easily accessible from winui code.
+
+
+# API Pages
+
+## Window.AppWindow property
+
+This newly introduced api will allow developers to use appwindow object directly within WinUI code.
+```c#
+public Microsoft.UI.Windowing.AppWindow Window.AppWindow { get; }
+```
+
+These *before* and *after* C# code examples illustrate how this api simplifies integrating appwindow apis in winui codebase.
+
+
+### Before
+```c#
+// This is needed to get any Window from inside a Xaml control 
+var xamlWindow = WindowHelper.GetWindowForElement(this /* some Xaml UIElement */);   
+
+// unnecessary boiler plate code 
+var windowId = Win32Interop.GetWindowIdFromWindow(WindowNative.GetWindowHandle(xamlWindow)); 
+var appWindow = AppWindow.GetFromWindowId(windowId); 
+
+//calling function foo  (not a real appwindow function)
+appWindow.foo();
+```
+
+### After
+```c#
+// This is needed to get any Window from inside a Xaml control 
+var xamlWindow = WindowHelper.GetWindowForElement(this /* some Xaml UIElement */);   
+
+//calling function foo directly
+xamlWindow.AppWindow.foo();  
+```
+Notice how `xamlWindow.AppWindow.foo()` doesn't require additional steps to call the function `foo`.
```

Comment:
Could you move this to the Background section? The way to think of this section is as the public docs, where we wouldn't want to include the old code.